src/core/fs.h

```diff
@@ -84,11 +90,23 @@ fs_basedir(
     size_t maxlen)  // allocation size
 {
 #ifdef __linux__
+  #ifdef LIBVKDT
+  void *handle;
+  char mod[PATH_MAX];
+  handle = dlopen("libvkdt.so", RTLD_LAZY);
+  if (!handle) {
+    fprintf(stderr, "%s\n", dlerror());
+	exit(1);
```

Comment:
ouch! i mean if we can't find the library then we're in deeper trouble than this :) but `exit()` is totally skipping any cleanup that may come in the main thread. i'm a bit afraid of what happens to gpus if you lock them up, and probably annoying if you're not writing config files or some such.

i'd just let the code fall through to whatever comes next and try to find a basedir the other way.

Reply:
Indeed.  Need to think through other ways, and/or return a result code from fs_basedir().